### PR TITLE
Added New Timeline

### PR DESCRIPTION
**Pull request by Seventy7dot2**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [ ] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [ ] Check the commit's or even all commits' message styles matches our requested structure.
- [ ] Check your code additions will fail neither code linting checks nor unit test.
- [ ] I have added necessary documentation (if appropriate)

### Description
Please describe your pull request.

❤️Thank you!

### Post merge checklist
- [ ] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

**Comment by ashutosh-rath02** [comment]
Attach a screen recoding of the component for multiple screens @Seventy7dot2 .

File: config/content/Timelinecontent.js
Comment (ashutosh-rath02) [inline config/content/Timelinecontent.js:1]: Remove the previous timeline component and change the file name

**Review by ashutosh-rath02 — CHANGES_REQUESTED** [review]

**Comment by Seventy7dot2** [comment]
> Attach a screen recoding of the component for multiple screens @Seventy7dot2 .


Uploading Screen Recording 2024-01-22 at 10.29.44 PM.mov…

**Comment by Seventy7dot2** [comment]
> Attach a screen recoding of the component for multiple screens @Seventy7dot2 .


https://github.com/dscnitrourkela/project-oregano/assets/114677481/d8c0c07b-7f1e-4855-96b5-6fcbe1bd7b6f

**Comment by ashutosh-rath02** [comment]
@Seventy7dot2 The animation is not smooth please look into it

**Comment by Seventy7dot2** [comment]
> @Seventy7dot2 The animation is not smooth please look into it

It is smooth. The problem in the video is cause of the responsive interface of chrome

**Comment by ashutosh-rath02** [comment]
@Shurtu-gal Can you provide the link to this preview?

**Comment by Shurtu-gal** [comment]
This has not been deployed.

**Review by Shurtu-gal — CHANGES_REQUESTED** [review]
Change base branch to `hacknitr5.0`.

**Comment by SirSimon162** [comment]
@Seventy7dot2 resolve conflicts

**Comment by Seventy7dot2** [comment]
> @Seventy7dot2 resolve conflicts

Done!👍

**Comment by Shurtu-gal** [comment]
@Seventy7dot2 open a new PR with branch `hacknitr5.0`.

**Pull request closed without merging**